test.js

```diff
@@ -1,29 +1,29 @@
 import test from 'ava';
 import Bluebird from 'bluebird';
-import m from './';
+import isPromise from '.';
```

Comment:
You told me that we should go for the module name instead. If you want me to revert it, let me know :).

Also, do you want me to fix the versions of AVA and XO?